test/parallel/test-async-wrap-final-did-throw.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const async_wrap = process.binding('async_wrap');
+var asyncThrows = 0;
+var ueCount = 0;
```

Comment:
Can you give this variable a more descriptive name?  It's not clear what 'ue' stands for.